examples/polypharmacy_gripnet/README.md

```diff
@@ -17,20 +17,28 @@ The original implementation of GripNet is [here](https://github.com/NYXFLOWER/Gr
 We train GripNet on three datasets constructed by [[2]](https://academic.oup.com/bioinformatics/article/34/13/i457/5045770?login=false).
 The datasets are downloaded from the [Bio-SNAP](http://snap.stanford.edu/biodata/) dataset collection. The integration of these datasets can be regarded as a knowledge graph, which contains the relations among proteins (P) and drugs (D) labeled with the P-P association, the P-D association and 1,317 side effects. The data statistics are shown below.
 
-| Dataset           | Nodes            | Edges   | #Unique Edge Labels |
-| ----------------- | ---------------- | ------- | ------------------- |
-| PP-Decagon        | 19,081(P)        | 715,612 | 1                   |
-| ChG-TargetDecagon | 3,648(P), 284(D) | 18,690  | 1                   |
-| ChChSe-Decagon    | 645(D)           | 63,473  | 1,317               |
+| Dataset           | Nodes            | Edges      | #Unique Edge Labels |
+| ----------------- | ---------------- | ---------- | ------------------- |
+| PP-Decagon        | 19,081(P)        | 715,612    | 1                   |
+| ChG-TargetDecagon | 3,648(P), 284(D) | 18,690     | 1                   |
+| ChChSe-Decagon    | 645(D)           | 4,649,441  | 1,317               |
 
 ## 4. Usage
 - Dataset download: [[here](https://github.com/pykale/data/tree/main/graphs)]
 
 - Algorithm: GripNet
 
-`python main.py --cfg configs/Drug_MINI-GripNet.yaml`
+```python
+python main.py
+```
 
-## References:
+We provide a `yaml` config file for a quick testing in the `configs` folder. To use it, run:
+
+```python
+python main.py --config configs/PoSE_MINI-GripNet.yaml
```

Comment:
This change from `cfg` to `config` is not necessary. We'd better be consistent over examples. If you'd like this change, you need to change all examples from cfg to config